### ISSUE TITLE
[BE] Ticket information from TalktixChatAPI to EmailProxy

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#41]
Expansion to: #38 #40 

- upon ticket creation send ticket information from TalktikChatAPI to EmailProxy 
- send email response with dead link to the created ticket as well as ticket information (content, subject line, ticket ID) via SMTP protocol
- also encapsulate proxy from Backend

## Acceptance criteria
- [x] The ticket information is received by the EmailProxy
- [x] The information is about the correct ticket
- [x] The email is send to the correct user
- [x] tests have been conducted

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by M-HRL** [comment]
![Image](https://github.com/amosproj/amos2023ws01-ticket-chat-ai/assets/60758809/591c1a01-57cb-490b-a233-f592b7cfb3ba)

**Comment by thogebati** [comment]
Both in 4 and 5
